docs/features/word-count.md

```diff
@@ -0,0 +1,108 @@
+---
+category: features
+---
+
+{@snippet features/build-word-count-source}
+
+# Word count
+
+The {@link module:wordcount/wordcount~WordCount} features provide a possibility to track the number of words and characters written in the editor.
+
+## Demo
+
+{@snippet features/word-count}
+
+```html
+<div id="editor">
+	<p>Hello world.</p>
+</div>
+<div class="word-count">
+</div>
+```
+
+```js
+ClassicEditor
+	.create( document.querySelector( '#editor' ), {
+		// configuration details
+	} )
+	.then( editor => {
+		const wordCountPlugin = editor.plugins.get( 'WordCount' );
+		const wordCountWrapper = document.querySelector( '.word-count' );
+
+		wordCountWrapper.appendChild( wordCounterPlugin.getWordCountContainer() );
+	} )
+	.catch( err => {
+		console.error( err.stack );
+	} );
+```
+
+## Configuring options
```

Comment:
If anything, it should be:

```suggestion
## Configuration options
```

Also, please check the structure of other feature guides. It's good to be consistent here.